src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEAutomatic.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const COEAutomatic = () => (
+  <>
+    <va-alert status="success">
+      <h3 slot="headline">Congratulations on your automatic COE</h3>
+      <div>
+        <p>
+          We have all the information we need, so you don’t need to fill out an
+          application. You can download your COE now.
+        </p>
+      </div>
+    </va-alert>
+    <div>
+      <h2>Review and download your COE</h2>
+      <p>
+        You can download your COE right now. If you need help, go to our
+        instructions for how to download and open a VA.gov PDF form.
+      </p>
+      <a href="#">Get instructions for downloading a VA.gov PDF</a>
```

Comment:
the href value here can be `/resources/how-to-download-and-open-a-vagov-pdf-form/`

Reply:
Updated